src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -64,30 +64,22 @@ Graph::~Graph() {
 }
 
 template<typename NET>
-void Graph::CreateGraph(NET &net, const GraphContext::CPtr ctx) {
+void Graph::CreateGraph(NET &model, const GraphContext::CPtr context) {
```

Comment:
@maxnick, as an option we could have just short names:
Create() {
  Init();
  Emit();
}
And, for now, have two options to create a graph: with a single 'create' call and more flexible 'init' and 'emit'
I expect that later, after we apply the same idea to the infer_request logic, we will drop a 'create' method and always require to use two stages initialization.